x-pack/plugins/security_solution/server/endpoint/routes/actions/list_handler.ts

```diff
@@ -74,6 +80,7 @@ export const actionListHandler = (
       }
 
       const requestParams = {
+        agentTypes: formatAgentTypeValues(agentTypes),
```

Comment:
Actually - one required change here in the route handler - you need to check the Feature Flag and Errror back if the agent type has `sentinel_one` in the value but the feature is off.

Reply:
FYI: I did something similar in the action create route handler as well

Reply:
oh yeah that's right! I'll update this before merging.

Reply:
updated c83b0677a7a6cd5fda0a20797651797b33797e7a